app/user/rest.py

```diff
@@ -392,7 +393,45 @@ def update_password(user_id):
     if errors:
         raise InvalidRequest(errors, status_code=400)
     update_user_password(user, pwd)
-    return jsonify(data=user_schema.dump(user).data), 200
+    return jsonify(data=user.serialize()), 200
+
+
+@user_blueprint.route('/<uuid:user_id>/organisations-and-services', methods=['GET'])
+def get_organisations_and_services_for_user(user_id):
+    user = get_user_and_accounts(user_id)
+    data = {
```

Comment:
It might be nice to pull this into a method.

Maybe it belongs in a schema or a method on the user?